changed fab option icons to fit more to overall app theme

pratik2315 opened the pull request:
## Pull Request template

## Purpose / Description
The FAB icons currently used do not necessarily represent interacting with a deck of cards (folder and + icon).

## Fixes
Fixes #9499 

## Approach
The icons now accurately represent the options whenever the user is interacting with them.

## How Has This Been Tested?

This has been tested on the Physical Device - Redmi 9 Prime.

## Learning (optional, can help others)


_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

![img1](https://user-images.githubusercontent.com/77961530/133031150-13c5b24f-4b27-4f02-bc35-b0216f063bce.jpeg)


https://user-images.githubusercontent.com/77961530/133031381-15f16e98-41fb-4c41-a5e0-2de2a7f0b14c.mp4

pratik2315 commented:
Please let me know if I made any mistake or any improvements I need to make.
Thanks for your time and support.

prateek-singh-3212 commented on AnkiDroid/src/main/res/layout/deck_picker.xml:
```diff
@@ -33,12 +33,12 @@
                         android:id="@+id/files"
                         android:layout_width="match_parent"
                         android:layout_height="match_parent"
+                        android:layout_above="@id/today_stats_text_view"
```
Please revert this change.

prateek-singh-3212 commented on AnkiDroid/src/main/res/layout/floating_add_button.xml:
```diff
@@ -131,7 +131,7 @@
                 android:layout_margin="10dp"
                 app:backgroundTint="?attr/fab_normal"
-                android:src="@drawable/ic_add_white"
+                android:src="@drawable/ic_add_card_filled_innerplus"
```
Please change the name of drawable to `ic_add_card_filled`. This avoids confusions in future.

prateek-singh-3212 reviewed (changes requested):
Thanks @pratik2315 for changing icons and Congratulations for your first PR. Few changes more and then I think we are good to go.

pratik2315 commented:
Hey @prateek-singh-3212 I have renamed the drawable accordingly and moved the "layout_above="@id/today_stats_text_view" to its original position and renamed the commit. pls check. thanks.

mikehardy commented on AnkiDroid/src/main/res/layout/floating_add_button.xml:
```diff
@@ -57,7 +57,7 @@
                 android:layout_height="wrap_content"
                 android:layout_margin="10dp"
                 app:backgroundTint="?attr/fab_normal"
-                android:src="@drawable/ic_folder_white"
```
fantastic! Thanks for posting this and thanks for giving it a review @prateek-singh-3212 !
I'm traveling and didn't have ankidroid git repo synced in my current laptop profile so I can't check quickly. @pratik2315 can you scan all files and make sure ic_folder_white / ic_add_white is still used ? If those files are no longer in use at all then I would like to see the files themselves removed as well - the codebase is big enough we do make an effort to purge things out immediately when not in use

mikehardy replied:
In fact the lint check fails for this reason on at least one file:

```

Lint found 1 error. First failure:
Fix the issues identified by lint, or add the following to your build script to proceed with errors:
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/drawable/ic_folder_white.xml:1: Error: The resource R.drawable.ic_folder_white appears to be unused [UnusedResources]
...
```

mikehardy reviewed (commented):
looks good - just a comment that it needs a sweep + possible purge of replaced icons if they are not in use any longer

mikehardy commented:
@pratik2315 if you could edit the description above and make sure it has the *exact* text `Fixes #issuenumber` then github crosslinks the issue and PR and will auto-close the issue when this is merged. You simply wrote `#issuenumber` which is not the magical way to do it for github cross-linking

mikehardy reviewed (changes requested):
lint not satisfied / unused file now after icon switch

pratik2315 commented:
hey @mikehardy and @prateek-singh-3212 thank you, sorry for the mess regarding such a simple issue I have deleted the unused drawable file and edited the description. pls check. thanks for your time

mikehardy commented:
No worries, this is a typical first PR, it's harder than it seems to get everything right. Wait till you work on a big feature! 😂😂

prateek-singh-3212 commented:
@pratik2315 No worries first PR is always harder and it teach you lots of thing about organization. Keep Going 🚀. 
And feel free to ask questions on discord.

david-allison-1 commented on AnkiDroid/src/main/res/drawable/ic_add_note.xml:
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
needs a copyright header. Should have been provided in the `.zip`

pratik2315 replied:
I could add it, but i dont hve the copyright header. I can add that if provided

david-allison-1 replied:
It's on the SVG in the original zip

pratik2315 replied:
Hey @david-allison-1 ! 
The files in the original zip are corrupted, they are not being loaded. I checked all of the other zip files sent on the issue, none of them have the header.

pratik2315 replied:
This page contains the following errors:
error on line 22 at column 6: XML declaration allowed only at the start of the document
Below is a rendering of the page up to the first error.

both of them have these errors

prateek-singh-3212 replied:
Header for file `add-deck-filled.svg`
```
Copyright 2021, Google Material Design Icons

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

Modifications (GitHub: joshakaw, 2021):
  Base icon: payments - outline from Google Material Icons
  Removed the circle from the middle of the rounded rectangle
  Added a plus icon inside the foreground bill.
```
Header for file `add-card-filled-innerplus.svg`

```
Copyright 2021, Google Material Design Icons

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

 Modifications (GitHub: joshakaw, 2021):
  Base icon: payments - outline from Google Material Icons
  Removed the circle from the middle of the rounded rectangle
  Removed the background outline of the 2nd card
  Added a plus icon, and removed, with padding, the area that the plus took up
```

pratik2315 replied:
Thanks @prateek-singh-3212 pls check

prateek-singh-3212 commented:
@pratik2315 Seems good now. @david-allison-1 please run the checks.

mikehardy reviewed (approved):
⚡ ! Nice, thank you

mikehardy commented:
Hi there! **September OpenCollective Notice**

Apologies for the delay, and if you're already submitted something to OpenCollective for PRs merged in **September** 2021, you may feel free to ignore this and it will be processed shortly

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for September 2021 submissions

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

(I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of August)

Thanks!
